helpers.sh

```diff
@@ -550,8 +546,10 @@ report_build_metrics() {
   local duration=$(($(timer_nanoseconds)-$smalltalk_ci_start_time))
   duration=$(echo "${duration}" | awk '{printf "%.3f\n", $1/1000000000}')
 
-  if [[ "${config_tracking}" != "true" ]]; then
-    return 0
+  if [ "${config_tracking+set}" ]; then
```

Comment:
Is this still needed?

Reply:
Not sure ... I [added it](https://github.com/dalehenrich/smalltalkCI/actions/runs/4654504882/jobs/8236306346#step:4:249) because a Moose build was failing, while the Squeak, Pharo, and GemStone builds were not failing ... and that "patch" addressed the problem ... I haven't tracked what you did to "solve" that particular problem in the general case, but if you think it's not needed I can certainly remove it or change it to the "right thing" ...

Reply:
I guess those were failing because the subshell that is supposed to upload coverage results failed. Let's revert this and see what the CI says :)

Reply:
will do ...

Reply:
Well, this one failed in `gtoolkit::prepare_gt`, not sure why though.